src/main/java/games/strategy/engine/framework/headlessGameServer/HeadlessServerSetup.java

```diff
@@ -42,11 +41,9 @@
   }
 
   void createLobbyWatcher() {
-    if (m_lobbyWatcher != null) {
-      m_lobbyWatcher.setInGameLobbyWatcher(InGameLobbyWatcher.newInGameLobbyWatcher(m_model.getMessenger(), null,
-          m_lobbyWatcher.getInGameLobbyWatcher()));
-      m_lobbyWatcher.setGameSelectorModel(m_gameSelectorModel);
-    }
+    m_lobbyWatcher.setInGameLobbyWatcher(InGameLobbyWatcher.newInGameLobbyWatcher(m_model.getMessenger(), null,
```

Comment:
If this class is no longer Serializable all of the fields could be renamed as well

Reply:
Agreed.  I actually have it on my Checkstyle TODO list.  I skipped it a few weeks ago because @DanVanAtta had already renamed these fields as part of #2509, which was subsequently closed (it was while reviewing that PR that I noticed the redundant `null` checks).  So, yes, it will be done in a follow-up PR. :smile: